app/assets/javascripts/app/review_form.js

```diff
@@ -134,11 +134,16 @@ App.addChild('ReviewForm', _.extend({
   unMaskFields: function(){
     this.$phone.unmask();
     this.$zip.unmask();
+    this.$cpf.unmask();
   },
 
   maskFields: function(){
     this.$phone.mask('(99)9999-9999Z', { translation:  {'Z': { pattern: /[0-9]/, optional: true }}});
     this.$zip.mask('99999-999');
+    this.$cpf.mask('999.999.999-99');
+    this.$('#payment_card_phone').mask('(99) 9999-99999');
```

Comment:
essa máscara aqui deveria ser igual a do outro telefone como te falei 
`this.$phone.mask('(99)9999-9999Z', { translation:  {'Z': { pattern: /[0-9]/, optional: true }}});`